data/map.txt

```diff
@@ -30168,10 +30168,12 @@ planet Luna
 	shipyard "Navy Basics"
 	shipyard "Betelgeuse Basics"
 	shipyard "Syndicate Basics"
+	shipyard description "Don't call the Luna shipyard workers Lunatics. They seem to be very sensitive about it."
 	outfitter "Common Outfits"
 	outfitter "Lovelace Basics"
 	outfitter "Syndicate Basics"
 	outfitter "Ammo North"
+	outfitter description "Don't call the Luna outfitter workers Lunatics. They seem to be very sensitive about it."
```

Comment:
normally I'd say to not include data changes but this is too good to remove. keep it!

Reply:
I think that's exactly why it's not really a great addition here.

Not only have you used basically the same text in the shipyard and outfitter, but is the implication that someone has put signs up around the shipyard and outfitter that say this? It makes sense as a hail or even as a news item, but it seems a bit silly here.

And, the fact that it is copied from the hail and reused in both the shipyard and outfitter looks extremely lazy and low quality.

Reply:
Ok will move it to just the build branch I have and link that build tag here.